src/pages/instances/Events.tsx

```diff
@@ -26,6 +28,8 @@ const Events: FC = () => {
       eventCallback.onFinish?.();
       eventQueue.remove(event.metadata.id);
     }
+
+    void refetchOperations();
```

Comment:
```suggestion
```
We shouldn't need to refetch again, we already do that below. This results in a lot of additional requests.